### PR TITLE
Riot shotguns for cargo : Revised Edition 2019 GOTY

### PR DESCRIPTION
**Pull request by Oshibkka**
## About The Pull Request
Adds riot shotgun crates to cargo.

## Why It's Good For The Game
Allows a cheaper and more practicable alternative to combat shotguns.  Helps with the problem of running out of shotguns during war ops since only 3 or 4 spawn in the armory roundstart.

## Changelog
:cl:
add: allows cargo to purchase riot shotgun crates
/:cl:

### CODE REVIEW

**Comment by Crossedfall** [comment]
Can't even compile due to indentation errors. Please fix the errors and make sure you are testing this locally to ensure that it compiles.

**Comment by Oshibkka** [comment]
Weird. When I compiled it there were no errors. Will look into it tomorrow

**Comment by Oshibkka** [comment]
Alright, I fixed it @Crossedfall

**Review by MarkSuckerberg — APPROVED** [review]
Seems good to me.

**Comment by Kerbin-Fiber** [comment]
but can't you just buy this crate and then sell the shotguns to centcomm for a cargo bounty?

Edit: the bounty does not exceed the price of this crate

**Comment by Oshibkka** [comment]
Yeah, I increased the price for that reason.

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -484,6 +484,20 @@
 					/obj/item/storage/belt/bandolier)
 	crate_name = "combat shotguns crate"
 
+/datum/supply_pack/security/armory/riot_shotgun_single
+	name = "Riot Shotgun Single-Pack"
+	desc = "When the clown's slipped you one times too many. Requires armory access to open."
```
Comment (qwertyquerty) [inline code/modules/cargo/packs.dm]: Fix the grammer "one times too many"

**Pull request merged**